packages/terra-compact-interactive-list/src/proptypes/columnShape.js

```diff
@@ -48,6 +48,10 @@ const columnShape = PropTypes.shape({
     alignTypes.LEFT,
   ]),
 
+  /**
+   * Boolean value indicating whether or not the cells in this column are selectable.
+   */
+  isSelectable: PropTypes.bool,
```

Comment:
I would expect the isSelectable prop to be on the cells themselves.  What if the consumer doesn't want every cell in the column to behave the same way?

Reply:
Currently it is determined for the whole column rather than each cell and included in the column shape. The cell selection mechanism might be a subject to change in the future, current implementation might change in the future as it wasn't completely clear what is the best way for it. Improving that is out of scope of this JIRA and rather a subject for the future improvements, as it needs discussion with @eawww